app/models/course/assessment.rb

```diff
@@ -142,7 +142,12 @@ def satisfiable?
     published?
   end
 
-  # The password to preventing attempting submission from multiple sessions.
+  # The password to prevent attempting submission from viewing the assessment.
+  def password_protected?
```

Comment:
Perhaps name the method as `view_password_protected?` for consistency?

Reply:
will do, thanks!